core/src/main/scala/filodb.core/store/IngestionConfig.scala

```diff
@@ -49,7 +48,6 @@ final case class StoreConfig(flushInterval: FiniteDuration,
     ConfigFactory.parseMap(Map("flush-interval" -> (flushInterval.toSeconds + "s"),
                                "time-aligned-chunks-enabled" -> timeAlignedChunksEnabled,
                                "disk-time-to-live" -> (diskTTLSeconds + "s"),
-                               "demand-paged-chunk-retention-period" -> (demandPagedRetentionPeriod.toSeconds + "s"),
```

Comment:
Looks like `demand-paged-chunk-retention-period` is referred in few other places too, could you please grep and clean the same.

Reply:
good catch! thanks! Will change